tests/NSubstitute.Acceptance.Specs/NSubstitute.Acceptance.Specs.csproj

```diff
@@ -1,17 +1,17 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
**thought:** I tried [porting over a few relevant tests](https://github.com/nsubstitute/NSubstitute/pull/699) from the old unit test project ages ago. I wonder if we should just give up on that and delete the old specs code?

Reply:
Unfortunately, I'm not in context of history for this project. 

My private opinion:
For me looks reasonable "save as much as we can" and delete it